### PR TITLE
Added one-off script to back populate hoc_survey_prize purpose for Hoc2014

### PR DESCRIPTION
**Pull request by aoby**

### CODE REVIEW

File: bin/oneoff/back_populate_hoc_survey_prize_purpose [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+# This script populates the purpose field in Pegasus.hoc_survey_prizes for claimed prizes with no purpose to "Hoc2014"
+
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+
+DB[:hoc_survey_prizes].exclude(claimant: nil).where(purpose: '').update(purpose: 'Hoc2014')
```
Comment (ashercodeorg) [inline bin/oneoff/back_populate_hoc_survey_prize_purpose:7]: Why are we excluding the rows where `claimant` is `nil`?

Comment (aoby) [reply bin/oneoff/back_populate_hoc_survey_prize_purpose:7]: Those are unclaimed prizes in the pool. They don't have a purpose until they're [claimed](https://github.com/code-dot-org/code-dot-org/blob/66d3bb58af12e0c89a95329a7c63a7141c243c43/pegasus/helpers/prize_helpers.rb#L7-L7)

Comment (ashercodeorg) [reply bin/oneoff/back_populate_hoc_survey_prize_purpose:7]: Fair enough. Though I'll continue to disagree (with existing code). Not having a purpose (until claimed) suggests the code could / would be used for anything other than a specific HOC.

**Comment by aoby** [comment]
Yes, I tested locally with an unclaimed prize, a claimed (Hoc2014) prize with no purpose, and a Hoc2015 claimed prize.

File: bin/oneoff/back_populate_hoc_survey_prize_purpose (hunk 1 above)
Comment (aoby) [reply bin/oneoff/back_populate_hoc_survey_prize_purpose:7]: How would you prefer it? Require a purpose for every prize and only allow claiming one if the purpose matches?

Note - currently they're not wired up to be claimed by anything. If we prefer this (or another approach), then when (if) we ramp up to use this table again on the next HOC we can make the change.

**Pull request merged**